socorro/external/fs/crashstorage.py

```diff
@@ -2,26 +2,22 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
+from contextlib import contextmanager, closing
 import gzip
 import json
 import os
 
-from contextlib import contextmanager, closing
-
-try:
-    from cStringIO import StringIO
-except ImportError:
-    from StringIO import StringIO
-
 from configman import Namespace
+from six import BytesIO
```

Comment:
This is tricky. In Python2, this ends up being StringIO and in Python3, this ends up being BytesIO.